types/nightwatch/test/testClientCommands.ts

```diff
@@ -297,6 +297,16 @@ describe('resizeWindow command demo', function() {
     });
 });
 
+//
+// .Screenshot
+//
+describe('screenshot command demo', function() {
+    test('async demo test', async function(browser) {
+        const result = await browser.screenshot(true);
+        isType<string>(result);
+    });
+});
```

Comment:
Can you please add a test for the sync way of using `.screenshot()` as well, with a callback?